common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -401,6 +435,7 @@ export class SendTransaction extends React.Component {
     // TODO sub gas for eth
     if (value === 'everything') {
       if (unit === 'ether') {
+        // TODO - why do we never use this?
```

Comment:
We do, line 453, we set value to state. I think this function triggers when you hit the "Send whole balance" button. But I also think this is flawed, since it doesn't account for gas. Might be best to just remove this for now.

Reply:
I'll actually go ahead and just finish off send whole balance now.

Reply:
What a champ 💪